src/pages/Categories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useDispatch } from 'react-redux/es/exports';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+  const CHECK_STATUS = 'book-store/categories/CHECK_STATUS';
+  return (
+    <>
+      <button type="button" onClick={() => dispatch({ type: CHECK_STATUS })}>
+        Check Status
+      </button>
```

Comment:
Kindly make sure that the `Under Construction` text is displayed when I click the `check status` button.

Hint: You can achieve this by using `useSelector` to fetch the categories state.
i.e
```
import { useSelector } from 'react-redux';

const status = useSelector((state) => state.category);

return (
    <p>{status}</p>
)
```